<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMaterialDialog</class>
 <widget class="QDialog" name="NewMaterialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New material</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMaterialName">
     <item>
      <widget class="QLabel" name="labelMaterialName">
       <property name="text">
        <string>Material name</string>
       </property>
       <property name="buddy">
        <cstring>lineEditMaterialName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMaterialName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidgetLinearElastic">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLinearElastic">
      <attribute name="title">
       <string>Linear elastic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutYoung">
         <item>
          <widget class="QLabel" name="labelYoung">
           <property name="text">
            <string>E</string>
           </property>
           <property name="buddy">
            <cstring>lineEditYoung</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditYoung"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutPoisson">
         <item>
          <widget class="QLabel" name="labelPoisson">
           <property name="text">
            <string>Î½</string>
           </property>
           <property name="buddy">
            <cstring>lineEditPoisson</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPoisson"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditMaterialName</tabstop>
  <tabstop>lineEditYoung</tabstop>
  <tabstop>lineEditPoisson</tabstop>
  <tabstop>tabWidgetLinearElastic</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewMaterialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewMaterialDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
